src/main/java/de/retest/recheck/RecheckImpl.java

```diff
@@ -127,11 +127,18 @@ public void capTest() {
 		currentTestResult = null;
 		final Set<Object> uniqueDifferences = finishedTestResult.getUniqueDifferences();
 		if ( !uniqueDifferences.isEmpty() ) {
-			throw new AssertionError( "\nA detailed report will be created at \'" + getResultFile() + "\'. " //
-					+ "You can review the details by using our GUI from https://retest.de/review/." //
-					+ "\n\nThe following differences have been found in \'" + suiteName //
-					+ "\'(with " + finishedTestResult.getActionReplayResults().size() + " check(s)):" //
-					+ "\n" + finishedTestResult.toString() );
+			final String message;
```

Comment:
How about two `private` methods, e.g., `getNoRecheckFilesErrorMessage()` and `getDifferencesErrorMessage()`. Then we could do something like:

```java
String msg = finishedTestResult.hasNoRecheckFiles() ? getNoRecheckFilesErrorMessage() : getDifferencesErrorMessage();
throw new AssertionError( msg );
```

Reply:
Good idea.